Core.DomainModel/Organization/Organization.cs

```diff
@@ -145,5 +153,55 @@ public Maybe<OrganizationUnit> GetOrganizationUnit(Guid organizationUnitId)
         {
             return OrgUnits.FirstOrDefault(unit => unit.Uuid == organizationUnitId);
         }
+
+        public Maybe<UIModuleCustomization> GetUiModuleCustomization(string module)
+        {
+            if (module == null)
+                throw new ArgumentNullException(nameof(module));
+            
+            return UIModuleCustomizations
+                .SingleOrDefault(config => config.Module == module)
+                .FromNullable();
+        }
+
+        public Result<UIModuleCustomization, OperationError> ModifyModuleCustomization(string module, IEnumerable<CustomizedUINode> nodes)
+        {
+            var uiNodes = nodes.ToList();
+            var customizedUiNodes = uiNodes.ToList();
+
+            var keysValidity = CheckKeysValidity(uiNodes);
+            if(keysValidity.HasValue)
+                return new OperationError(keysValidity.Value, OperationFailure.BadInput);
+
+            var moduleCustomization = GetUiModuleCustomization(module).GetValueOrDefault();
+            if (moduleCustomization == null)
+            {
+                moduleCustomization = new UIModuleCustomization { OrganizationId = Id, Module = module };
```

Comment:
set the Organization property in stead (this `this`) It's more natural and less db